gateway-service/src/chef/chef.service.ts

```
@@ -0,0 +1,89 @@
+      throw error;
+    }
+  }
+
+  async getProfileChef(id: number): Promise<any> {
+    try {
+      const result = await lastValueFrom(
+        this.chefClient.send('chef/profile', id),
+      );
+      return result;
+    } catch (error) {
+      throw error;
```

Biome

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)